backend.js

```diff
@@ -11,6 +11,18 @@ var tables = [];
 
 var waitList = [];
 
+app.get("/", function(req, res) {
+    res.sendFile(path.join(__dirname, "home.html"));
+  });
+
+  app.get("/api/tables", function(req, res) {
```

Comment:
This path should just be "/tables" to get to the tables.html file.  The path "/api/tables" will take the user to an empty html page that returns the tables array.